src/google/protobuf/arena.h

```diff
@@ -153,6 +153,23 @@ struct ArenaOptions {
         on_arena_destruction(NULL),
         on_arena_allocation(NULL) {}
 
+  ArenaOptions(size_t start_block_size,
+               size_t max_block_size,
+               char* initial_block,
+               size_t initial_block_size,
+               void* (*block_alloc)(size_t),
+               void (*block_dealloc)(void*, size_t),
```

Comment:
I think instead of a comma there needs to be a closing parenthesis at the end of this line.